src/services/PugQueue.js

```diff
@@ -25,36 +28,37 @@ export class PugQueue {
     this.queue.push(member)
     member.addRole(process.env.PUGS_ROLE)
 
-    return `${String.fromCodePoint(0x2705)} ${name} added to queue. ${this.getQueueState()}`
+    return `${String.fromCodePoint(0x2705)} ${name} ${pug_roles} added to queue. ${this.getQueueState()}`
```

Comment:
added them in there so that it's immediately clear as what you are queued as. perhaps people want to queue specifically as a tank, but next game they dont want to tank or whatever. just makes it easy to see what roles you have.